apps/e2e/datetime-storybook/src/app/date-range-picker/date-range-picker.component.ts

```diff
@@ -15,11 +15,13 @@ export class DateRangePickerComponent {
   public scenarios: {
     name: string;
     label: string;
+    hintText?: string;
```

Comment:
Minor - normally we would place optional parameters after non-optional ones. So this should probably go after `calculatorIds`.